### DISCUSSION TITLE
[Feature]: Archive section for assets

### DISCUSSION THREAD

**Discussion by zackpollard** [immich-app/immich#1681] [Feature Request]
### Feature detail

Sometimes photos are uploaded that you don't want to keep in your main gallery, but might still want backed up. Examples of this would include receipts, paperwork, menus, screenshots, etc. We should have a way to select photos and archive them and include a section on the sidebar to view that archive section.

### Platform

Web & Mobile

**Comment by jrasm91** [comment]
Implemented in [v1.54.](https://github.com/immich-app/immich/releases/tag/v1.54.0)

**Comment by TheOriginal92** [comment]
@jrasm91 Thank you for implementing this feature!

To improve on it, i would like to suggest adding a kind of password (using the same as for the account) to access archived media.
Similar to how its done on iPhones for example.

Thank you very much!

**Reply by jrasm91** [reply]
You can track that here: https://github.com/immich-app/immich/discussions/2451
